src/definitions/pinghf.json

```diff
@@ -0,0 +1,465 @@
+                "payload": [
+                    {
+                        "name": "ping_interval",
+                        "type": "u16",
+                        "description": "The interval between acoustic measurements.",
+                        "units": "ms"
+                    }
+                ]
+            },
+            "set_gain_setting": {
+                "id": "2005",
+                "description": "Set the current gain setting.",
+                "payload": [
+                    {
+                        "name": "gain_setting",
+                        "type": "u8",
+                        "description": "The current gain setting. The analog gain can be determined by requesting the analog_gain_levels message."
+                    }
+                ]
+            },
+            "set_ping_enable": {
+                "id": "2006",
+                "description": "Enable or disable acoustic measurements.",
+                "payload": [
+                    {
+                        "name": "ping_enabled",
+                        "type": "u8",
+                        "description": "0: Disable, 1: Enable."
+                    }
+                ]
+            },
+            "set_auto_gain": {
+                "id": "2007",
+                "description": "Enable or disable automatic gain adjustments.",
+                "payload": [
+                    {
+                        "name": "enabled",
+                        "type": "u8",
+                        "description": "0: Disable, 1: Enable."
+                    }
+                ]
+            },
+            "set_auto_range": {
+                "id": "2008",
+                "description": "Enable or disable automatic scan range adjustments.",
+                "payload": [
+                    {
+                        "name": "enabled",
+                        "type": "u8",
+                        "description": "0: Disable, 1: Enable."
+                    }
+                ]
+            },
+            "set_num_profile_data_points": {
+                "id": "2009",
+                "description": "Set the number of data points to be returned in profile2 messages.",
+                "payload": [
+                    {
+                        "name": "num_points",
+                        "type": "u16",
+                        "description": "new length for the data array in profile2 messages"
+                    }
+                ]
+            }
+        },
+        "get": {
+            "device_id": {
+                "id": "2201",
+                "description": "The device ID.",
+                "payload": [
+                    {
+                        "name": "device_id",
+                        "type": "u8",
+                        "description": "The device ID (0-254). 255 is reserved for broadcast messages."
+                    }
+                ],
+                "deprecated": "true"
```

Comment:
Should we have a deprecated function here ? We did not release yet.
Besides that, we don't have set_device_id anymore

Reply:
I'll take it out.